client/helpers/scenes.ts

```diff
@@ -54,82 +103,188 @@ export interface PollSceneReadyArgs extends BaseArgs {
   readonly polling?: Polling;
 }
 
+export interface UpdateSceneReq extends Base {
+  readonly attributes: UpdateSceneRequestDataAttributes;
+}
+
+export interface UpdateSceneRes {
+  scene: Scene;
+}
+
+interface Base {
+  /** An instance of {@link VertexClient}. */
+  readonly client: VertexClient;
+
+  /** ID of scene. */
+  readonly sceneId: string;
+}
+
+interface QueuedSceneItem {
+  failure?: Failure;
+  job?: QueuedJob;
+  req: CreateSceneItemRequest;
+}
+
 /**
  * Create a scene with scene items.
- *
- * @param args - The {@link CreateSceneWithSceneItemsArgs}.
  */
-export async function createSceneWithSceneItems({
+export async function createSceneAndSceneItems({
```

Comment:
Created a new method that has dedicated req/res object like we do with gRPC. This allows us to add to them without breaking clients.